Cubical/Foundations/Equiv.agda

```diff
@@ -71,3 +87,13 @@ compEquiv f g = isoToEquiv
 --   invEquivInvol f i .fst = fst f
 --   invEquivInvol f i .snd = propIsEquiv (fst f) (snd (invEquiv (invEquiv f))) (snd f) i
 
+-- Transport is an equivalence
+isEquivTransport : ∀ {ℓ} {A B : Set ℓ} (p : A ≡ B) → isEquiv (transport p)
```

Comment:
This is and the next result is in: https://github.com/agda/cubical/blob/master/Cubical/Foundations/Transport.agda#L28

(with better proofs)